src/parser/demonhunter/havoc/modules/features/Checklist/Component.js

```diff
@@ -99,10 +137,10 @@ class HavocDemonHunterChecklist extends React.PureComponent {
         </Rule>
 
         <Rule
-          name="Manage your resources/fury properly"
+          name="Manage your Fury properly"
           description={(
             <>
-            <p>You should always avoid capping your fury and spend it regularly.</p>
+            <p>You should always avoid capping your Fury and spend it regularly.</p>
```

Comment:
I saw Zero point out `<p>` tag in the previous Rule, I guess it shouldn't be here either?
```suggestion
            You should always avoid capping your Fury and spend it regularly.
```

Reply:
removed